File: src/codes/Users_Manual/modulation/fourier_0.rs
use autd3::modulation::{Fourier, FourierOption};
# use autd3::prelude::*;

# fn main() {
# let _ = 
Fourier {
    components: vec![
        Sine {
            freq: 100 * Hz,
            option: Default::default(),
        },
        Sine {
            freq: 150 * Hz,
            option: Default::default(),
        },
    ],
    option: FourierOption {
        scale_factor: None,
        clamp: false,
        offset: 0x00,
    },
};
# }

File: src/codes/Users_Manual/modulation/fourier_0.cpp
//~#include<autd3.hpp>
#include <autd3/modulation/fourier.hpp>

//~int main() {
//~using namespace autd3;
modulation::Fourier({Sine(100 * Hz, SineOption{}),
                     Sine(150 * Hz, SineOption{})},
                    modulation::FourierOption{
                        .scale_factor = std::nullopt,
                        .clamp = false,
                        .offset = 0x00,
                    });

//~return 0; }